site/src/lib/blocks.ts

```diff
@@ -20,13 +21,7 @@ export type BuildConfig = {
   timestamp?: string | null;
 };
 
-const getBlockMediaUrl = (
-  mediaPath: string | undefined | null,
-  packagePath: string,
-): string | null => {
-  if (!mediaPath) {
```

Comment:
`icon` and `image` SHOULD be specified in block metadata.

Reply:
SHOULD doesn't mean MUST - they might be absent. Please revert this. See https://www.ietf.org/rfc/rfc2119.txt

Reply:
Reverted